### DISCUSSION TITLE
Multi-PV evaluation glitch in SF

### DISCUSSION THREAD

**Discussion by ddugovic** [lichess-org/fishnet#179] [Ideas]
[official-stockfish Multi-PV slowly finds multiple checkmates](https://github.com/official-stockfish/Stockfish/issues/3406).  This causes Lichess to incorrectly flag checkmating moves as "inaccurate":

```
setoption name MultiPV value 2
position fen 3r2k1/1R3R1p/5p2/3p2pP/8/7K/PPr5/8 w - - 2 41
go infinite
...
info depth 42 seldepth 14 multipv 1 score mate 7 nodes 16461262 nps 1317111 hashfull 1000 tbhits 0 time 12498 pv f7g7 g8f8 g7h7 g5g4 h3g3 c2c7 b7c7 f8g8 c7g7 g8f8 h5h6 d8d7 h7h8
info depth 41 seldepth 48 multipv 2 score cp 7400 nodes 16461262 nps 1317111 hashfull 1000 tbhits 0 time 12498 pv h5h6 c2c4 h3g3 c4h4 f7g7 g8f8 g7h7 h4h6 h7h6 f8g8 g3g4 f6f5 g4f5 g5g4 b7h7 g8f8 h7h8 f8e7 h6h7 e7d6 h8d8 d6c5 f5g4 c5d4 h7c7 d4e4 c7d7 e4d3 a2a4 d3c2 d7d5 c2b1 a4a5 b1c2 a5a6 c2c1 a6a7 c1b1 a7a8q b1c2
```

![image](https://user-images.githubusercontent.com/1834123/111839148-57095c00-88c8-11eb-8a30-6fabb8617bfd.png)

**Comment by niklasf** [comment]
So the idea is the search with MultiPV? Wouldn't this make most analysis worse?

**Reply by ddugovic** [reply]
I had intended to suggest disabling of multi-PV search because I forgot we already disable it.  So I suppose the Stockfish regression (that suddenly it is much slower at finding checkmates than it used to be) occurs:
1. however it is that fishnet is already running SF
2. in my testing (multi-PV `go infinite` with 1 thread)
3. in my testing (multi-PV `go depth` with 1 thread)
4. in my testing (multi-PV `go mate` with 1 thread)
